docs/data/sdks/javascript/index.md

```diff
@@ -18,7 +18,14 @@ This is the official documentation for the Amplitude Analytics JavaScript SDK.
 --8<-- "includes/ampli-vs-amplitude.md"
     Click here for more documentation on [Ampli for Browser](./ampli.md).
 
---8<-- "includes/sdk-browser-supported-version.md"
+!!! warning "Supported browser versions"
```

Comment:
will we not fix the build for amplitude-js?

Reply:
I agree would be good to fix this.